improver/cli/estimate_emos_coefficients.py

```diff
@@ -112,5 +112,5 @@ def process(*cubes: cli.inputcube,
     return EstimateCoefficientsForEnsembleCalibration(
         distribution, cycletime, desired_units=units,
         predictor=predictor, tolerance=tolerance,
-        max_iterations=max_iterations).process(
+        max_iterations=max_iterations)(
```

Comment:
Long init list - could you split this into `plugin =  ...` and `return plugin.process(...` please.

Reply:
Done (without the .process obviously).